docs/billing.md

```diff
@@ -4,9 +4,16 @@ The GCP Service Broker automatically labels supported resources with organizatio
 
 When these supported services are provisioned, they will have the following labels populated with information from the request:
 
- * `pcf-organization-guid`
- * `pcf-space-guid`
- * `pcf-instance-id`
+* `cf-organization-guid`
+* `cf-organization-name`
+* `cf-space-guid`
+* `cf-space-name`
+* `instance-id`
+* `instance-name`
+* `k8s-namespace`
+* `k8s-clusterid`
```

Comment:
Shouldn't `managed-by` also be in this list?

Reply:
The `managed-by` is a constant which is why I pushed it into its own section these are all populated from the request.